dspace/config/modules/authentication-shibboleth.cfg

```diff
@@ -38,7 +38,7 @@
 authentication-shibboleth.lazysession = true
 
 # The url to start a shibboleth session (only for lazy sessions)
-authentication-shibboleth.lazysession.loginurl = /Shibboleth.sso/Login
+authentication-shibboleth.lazysession.loginurl = ${dspace.baseUrl}/Shibboleth.sso/Login
```

Comment:
Please note that this property it's about to be changed to: dspace.server.url

Reply:
We have to be careful which property is used here. It should not redirect to /server/Shibboleth.sso/Login (which it currently does in my installation)

I'd recommend to revisit this once https://github.com/DSpace/DSpace/pull/2657 is merged because there's a risk the URL is invalid

Reply:
@atarix83 : I'm unclear why this change is needed.  If we leave this setting as it was (`/Shibboleth.sso/Login`), that should cause the redirect to be to the servlet container root path.  That means if you were running DSpace REST API at http://localhost:8080/server/ then the path "/Shibbboleth.sso/Login" should cause the redirect to be to http://localhost:8080/Shibboleth.sso/Login.  

I believe that's how this setting worked in DSpace 6 and it also is how relative paths are described to work in HttpServletResponse.sendRedirect(): https://tomcat.apache.org/tomcat-8.5-doc/servletapi/javax/servlet/http/HttpServletResponse.html?is-external=true#sendRedirect(java.lang.String)  (This is the method that ultimately gets called with this relative URL)

So, I guess I'm asking why we need to turn this configuration into an absolute/full URL, as it worked previously as a relative URL?

Reply:
In DSpace 7, the difference is that the hostname of REST and Angular may differ
When logging in from the Angular UI, Angular should link to https://dspace7-rest/Shibboleth.sso/Login (not https://dspace7-rest/server/Shibboleth.sso/Login nor https://dspace7-angular/Shibboleth.sso/Login)

At this point in time, this would requiring taking ${dspace.server.url}, stripping the context path off, and adding /Shibboleth.sso/Login to it. That makes it hard to configure